app/helpers/standard_form_builder.rb

```diff
@@ -287,7 +287,8 @@ def has_many_field(attr, html_options = {}) # rubocop:disable Naming/PredicateNa
     ].compact.join(" ")
     html_options[:class] += " is-invalid" if errors_on?(attr)
 
-    add_css_class(html_options, "multiselect")
+    add_css_class(html_options, "multiselect tom-select")
+    html_options[:data] = {chosen_no_results: I18n.t("global.chosen_no_results")}
```

Comment:
`html_options[:data] = {chosen_no_results: I18n.t("global.chosen_no_results")}` doesn't seem to have any effect. Also I can't see the message `chosen_no_results: Keine Einträge gefunden mit` from the locale anywhere if it is set. Where should it show up?

Reply:
It is used when the search is enabled for a multiselect, and no results are found, like in this Case:
![image](https://github.com/user-attachments/assets/e530b62d-dd5b-4ccc-acf3-f4c0423eb97e)

But since we do not enable the search per default in this case anyways, you are correct, I can remove it :upside_down_face: